test/_utils.js

```diff
@@ -146,6 +146,28 @@ function sortApmEvents(events) {
     });
 }
 
+/**
+ * Slurp everything from the given ReadableStream and return the content,
+ * converted to a string.
+ */
+async function slurpStream(stream) {
```

Comment:
Note for reviewer: moved this function to a common place for further use in other client tests